config/config.go

```
@@ -0,0 +1,87 @@
+package config
+
+import (
+	"github.com/bianjieai/irita-sync/libs/logger"
+	"github.com/bianjieai/irita-sync/utils"
+	"github.com/bianjieai/irita-sync/utils/constant"
+	"github.com/spf13/viper"
+	"os"
+)
+
+var (
+	ConfigFilePath string
+)
+
+type (
+	Config struct {
+		DataBase DataBaseConf `mapstructure:"database"`
+		Server   ServerConf   `mapstructure:"server"`
+	}
+
+	DataBaseConf struct {
+		Addrs    string `mapstructure:"addrs"`
+		User     string `mapstructure:"user"`
+		Passwd   string `mapstructure:"passwd" json:"-"`
+		Database string `mapstructure:"database"`
+	}
+
+	ServerConf struct {
+		NodeUrls                  string `mapstructure:"node_urls"`
+		WorkerNumCreateTask       int    `mapstructure:"worker_num_create_task"`
+		WorkerNumExecuteTask      int    `mapstructure:"worker_num_execute_task"`
+		WorkerMaxSleepTime        int    `mapstructure:"worker_max_sleep_time"`
+		BlockNumPerWorkerHandle   int    `mapstructure:"block_num_per_worker_handle"`
+		SleepTimeCreateTaskWorker int    `mapstructure:"sleep_time_create_task_worker"`
+
+		MaxConnectionNum   int    `mapstructure:"max_connection_num"`
+		InitConnectionNum  int    `mapstructure:"init_connection_num"`
+		Bech32ChainPrefix  string `mapstructure:"bech32_chain_prefix"`
+		ChainId            string `mapstructure:"chain_id"`
+		ChainBlockInterval int    `mapstructure:"chain_block_interval"`
+		BehindBlockNum     int    `mapstructure:"behind_block_num"`
+
+		PromethousPort    string `mapstructure:"promethous_port"`
+		OnlySupportModule string `mapstructure:"only_support_module"`
+	}
+)
+
+func init() {
+	websit, found := os.LookupEnv(constant.EnvNameConfigFilePath)
+	if found {
+		ConfigFilePath = websit
+	} else {
+		ConfigFilePath = `D:\bianjie\dev\irita-sync\config\config.toml`
+		//panic("not found CONFIG_FILE_PATH")
```

should panic if not found CONFIG_FILE_PATH